onnxmltools/convert/coreml/operator_converters/neural_network/InnerProduct.py

```diff
@@ -6,32 +6,41 @@
 
 from .....proto import onnx_proto
 from ....common._registration import register_converter
-
+from ....common._apply_operation import apply_reshape
 
 def convert_inner_product(scope, operator, container):
     params = operator.raw_operator.innerProduct
-    op_type = 'FC'
+    op_type = 'Gemm'
     inputs = [variable.full_name for variable in operator.inputs]
```

Comment:
Do you want to use `inputs = []`? So that the following `inputs[0] = reshaped_tensor_name ` can be changed to `inputs.append(reshaped_tensor_name) ` for higher consistency.